tools/plotjuggler/juggle.py

```diff
@@ -69,22 +76,22 @@ def juggle_route(route_name, segment_number, qlog):
   save_log(tempfile.name, all_data, compress=False)
   del all_data
 
-  juggle_file(tempfile.name, dbc)
+  juggle_file(tempfile.name, dbc, bin_path)
 
 def get_arg_parser():
   parser = argparse.ArgumentParser(description="PlotJuggler plugin for reading rlogs",
                                    formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
   parser.add_argument("--qlog", action="store_true", help="Use qlogs")
+  parser.add_argument("--bin_path", nargs='?', help="Run plotjuggler using executable in bin_path (absolute path)")
```

Comment:
I'd just make this an env var called `PLOTJUGGLER_PATH` with a default of `plotjuggler`